### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by philipithomas**

### CODE REVIEW

File: upload.go [hunk 1]
```diff
@@ -34,15 +34,22 @@ func Upload(file *os.File) (string, error) {
 	}
 
 	cfg := aws.NewConfig().WithRegion(awsRegion).WithCredentials(creds)
-	svc := s3.New(session.New(), cfg)
+	sesh, err := session.NewSession()
```
Comment (philipithomas) [inline upload.go:37]: session.New is deprecated https://docs.aws.amazon.com/sdk-for-go/api/aws/session/

File: upload.go [hunk 2]
```diff
@@ -34,15 +34,22 @@ func Upload(file *os.File) (string, error) {
...
+	svc := s3.New(sesh, cfg)
 
 	fileInfo, err := file.Stat()
 	if err != nil {
 		log.Fatalf("%v", err)
 	}
 
 	buffer := new(bytes.Buffer)
-	file.Seek(0, 0)
+	_, err = file.Seek(0, 0)
```
Comment (srt32) [inline upload.go:49]: Excellent find to check the `err`. Is there a better way than using `Seek` here?

**Comment by srt32** [comment]
Thanks for the PR! There are currently no specs ;(. Are you able to confirm the session change works? I can test manually later, if no.

**Comment by srt32** [comment]
Works great! Thanks ‼️

**Pull request merged**